introducing-java/src/main/java/com/onto/javacourse/intro/gcd/GCD.java

```diff
@@ -1,5 +1,33 @@
-public class Main {
-    public static void main(String[] args) {
+public class GCD {
 
+    /**
+     * Method to find GCD
```

Comment:
The comment violates the java conventions. Method names start with a verb and end with a dot. I don't need to know from the comment that it is a method so this is redundant. For instance good comment would be `Calculates the greatest common divisor of two numbers.`.

Reply:
You could add to the comment that you are using Euclidean algorithm for instance.